paasta_tools/cli/cmds/local_run.py

```diff
@@ -74,20 +74,20 @@ def perform_http_healthcheck(url, timeout):
             try:
                 res = requests.get(url)
             except requests.ConnectionError:
-                return (False, "http request failed: connection failed")
+                return False, "http request failed: connection failed"
```

Comment:
I'm not sure how I feel about unwrapping these tuples. I don't think I feel strongly one way or another, but I'm slightly leaning towards leaving the parens on, because explicit is better than implicit.

I didn't see anything in pep8 about tuples/multiple return values and whether parens or no parens are preferred.